packages/docs/docs/theme/default-theme-config.md

```diff
@@ -409,17 +409,23 @@ module.exports = {
     docsBranch: 'master',
     // defaults to false, set to true to enable
     editLinks: true,
+    // default value is true. Allows to hide next page links on all pages
+    nextLinks: false,
+    // default value is true. Allows to hide prev page links on all pages
+    prevLinks: false,
     // custom text for edit link. Defaults to "Edit this page"
     editLinkText: 'Help us improve this page!'
   }
 }
 ```
 
-You can also hide the edit link on a specific page via `YAML front matter`:
+You can overwrite the following properties on specific pages via `YAML front matter`:
 
 ``` yaml
 ---
```

Comment:
It has already been mentioned in [the previous section](https://v1.vuepress.vuejs.org/theme/default-theme-config.html#prev-next-links).

Reply:
Yes but I think it's a good thing to also mention it here as users won't necessarily have read [this part](https://v1.vuepress.vuejs.org/theme/default-theme-config.html#prev-next-links) before.

Reply:
I mean maybe you can update `Prev / Next Links` to tell folks that they can disable next and prev links from global config ( Instead of updating `Git repository and Edit Links`). Just my opinion.

Reply:
Sure! I can update both parts indeed 😉

Reply:
Updated in 003f4e87ec71efd611a5ada4c9c246861077fb88